sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -157,6 +161,7 @@ def begin_recognize_receipts_from_url(self, receipt_url, **kwargs):
         """
 
         polling_interval = kwargs.pop("polling_interval", POLLING_INTERVAL)
+        continuation_token = kwargs.pop("continuation_token", None)
```

Comment:
Since you're just popping continuation_token and putting it back in, you should just not pop it at all, as the kwargs passed into the function call will have it

Reply:
I'd like to do that but in many of the methods I'm controlling the lro_algorithm so I need to pass in, e.g., LROBasePolling(**kwargs) myself into the generated call. I need to pop it here so it doesn't get passed into the polling method.